platform/lib/routers/packager.js

```diff
@@ -52,11 +53,14 @@ const packager = (request, response, next) => {
     next();
     return;
   }
-  // We'll only set Vary: AMP-Cache-Transform for html or SXG requests
-  const acceptHeader = request.header('accept');
-  if (acceptHeader &&
-        (!acceptHeader.includes('text/html') &&
-         !acceptHeader.includes('application/signed-exchange'))) {
```

Comment:
As far as I understand, the check for the accept header is mandatory here and should not be removed.
Or did I miss something here? I do not know much about this whole topic.

Reply:
You're right. It's not mandatory, but helps our CDN to cache more efficiently. This change to please toolbox-linter ([context](https://github.com/ampproject/amp-toolbox/issues/456)). 

However, thinking more about this, the correct approach would be to check that both `text/html` and `application/signed-exchange` are present at the **same** time. Only if both are present we need to vary on the `amp-cache-transform` header (otherwise, `vary: accept` is already sufficient for the CDN to differentiate the responses).

Reply:
Based on the discussing [here](https://github.com/ampproject/amp-toolbox/issues/456) and Devin's feedback we should always set the vary header as it's implemented in the PR.